src/App.js

```diff
@@ -11,29 +11,23 @@ import LoginPage from './pages/LoginPage/LoginPage';
 import SignUpPage from './pages/SignUpPage/SignUp';
 import CartPage from './pages/CartPage/CartPage';
 import { useAuth } from "./context/AuthContext";
-
+import { ROUTE_PATH_CartPage, ROUTE_PATH_LandingPage, ROUTE_PATH_LoginPage, ROUTE_PATH_Mockman, ROUTE_PATH_ProductListPage, ROUTE_PATH_SignupPage, ROUTE_PATH_Unknown, ROUTE_PATH_WishListPage } from "./utils/Routes"
 function App() {
   const { login, setlogin } = useAuth();
   return (
     <div className="App">
-     <Routes >
-       <Route element={<Main/>}>
-         <Route path='/' element={<LandingPage/>} />
-         <Route path="/products" element={<ProductListPage/>} />
-         <Route path="/wishlist" element={<WishListPage/>} />
-         <Route path="/cart" element={<CartPage/>} />
-         <Route path="/login" element={login ? <Navigate to="/" replace/> : <LoginPage/>} />
-         <Route path="/signup" element={login ? <Navigate to="/" replace/> : <SignUpPage />} />
-         <Route path="*" element={<LandingPage/>} />
-       </Route>
-       <Route path='/mockman' element={<div className='MockAPI'><Mockman/></div>}/>
-     </Routes>
-
-{/* 
-       <Navbar/>
-       <LandingPage/>
-        <ProductListPage/>
-      <Footer/> */}
+      <Routes >
+        <Route element={<Main />}>
+          <Route path={ROUTE_PATH_LandingPage} element={<LandingPage />} />
+          <Route path={ROUTE_PATH_ProductListPage} element={<ProductListPage />} />
+          <Route path={ROUTE_PATH_WishListPage} element={<WishListPage />} />
+          <Route path={ROUTE_PATH_CartPage} element={<CartPage />} />
+          <Route path={ROUTE_PATH_LoginPage} element={login ? <Navigate to="/" replace /> : <LoginPage />} />
```

Comment:
You can use private route here. Create a component called PrivateRoute and use the same logic there. A bit of googling would surely help

Reply:
okay got it but currently I have few important things left if time persist i will add them
I have looked into them 
but thnks for the comment